helpers/playerHelper.js

```diff
@@ -78,6 +87,7 @@ exports.parseCollection = function(players, subresources) {
       switch (resource) {
         case 'stats':
           player.stats = self.mapStats(p[idx + 1].player_stats);
+          player.points = self.mapPoints(p[idx + 1].player_points);
```

Comment:
Specifically for football. I don't play any other fantasy sports so I'm not sure - and I could not get the existing tests to run